RELEASE-NOTES.md

```diff
@@ -9,669 +12,343 @@
-```
-
-Note that the object path has changed from
-`path1%2Fpath2%2Fte%2Bst.txt` to `path1%2Fpath2%2Fte%252Bst.txt` between
-the two examples shown.
-
-If the old behavior is preferred, perhaps because
-applications using Riak CS have been written to use it, you can retain
-that behavior by modifying your Riak CS configuration upon upgrade.
-Change the `rewrite_module` setting as follows:
-
-```erlang
-{riak_cs, [
-    %% Other settings
-    {rewrite_module, riak_cs_s3_rewrite_legacy},
-    %% Other settings
-]}
-```
-
-**Note**: The old behavior is technically incorrect and implicitly
-overwrites data in the ways described above, so please retain the old
-behavior with caution.
+# General Notes on Upgrading to Riak CS 2.0
 
-# Riak CS 1.5.3 Release Notes
+Upgrading Riak CS system involves upgrading underlying Riak and
+Stanchion process. Upgrade process might be complicated depending on
+system configuration and version combination. This document describes
+general instructions of upgrading the whole system to Riak CS 2.0, and
+notices on upgrading.
 
-## Additions
+For minor use cases and more detail information, consult Riak and Riak
+CS's release notes.
 
-- Add read_before_last_manifest_write option to help avoid sibling
-  explosion for use cases involving high churn and concurrency on a
-  fixed set of keys. [riak_cs/#1011](https://github.com/basho/riak_cs/pull/1011)
-- Add configurable timeouts for all Riak CS interactions with Riak to
-  provide more flexibility in operational
-  situations. [riak_cs/#1021](https://github.com/basho/riak_cs/pull/1021)
+* [Riak 1.4 series release notes](https://github.com/basho/riak/blob/1.4/RELEASE-NOTES.md)
+* [Riak 2.0 series release notes](https://github.com/basho/riak/blob/2.0/RELEASE-NOTES.md)
+* [Riak CS 1.4 series release notes](https://github.com/basho/riak_cs/blob/release/1.5/RELEASE-NOTES.md#riak-cs-145-release-notes)
+* [Riak CS 1.5 series release notes](https://github.com/basho/riak_cs/blob/release/1.5/RELEASE-NOTES.md#riak-cs-154-release-notes)
 
-## Bugs Fixed
+## Upgrading From Riak CS < 1.4
 
-- Fix storage usage calculation bug where data for deleted buckets
-  would be included in the calculation
-  results. [riak_cs/#996](https://github.com/basho/riak_cs/pull/996)
+Basho only support 2 older versions from the latest release. For Riak
+CS 2.0 case, only upgrades from 1.4 series and 1.5 series are
+supported. Basically this pattern is out of the scope of this
+document.
 
-# Riak CS 1.5.2 Release Notes
+To upgrade to 2.0 from these versions, operators need another extra
+step to upgrade the whole system to Riak CS 1.4.5 or Riak
+CS 1.5.4. Upgrading to 1.5.4 would rather be recommented than
+to 1.4.5. Thus underlying Riak would also better upgraded to 1.4
+series - hopefully the latest 1.4.12.
 
-## Additions
+Notes TBD: any caveats on upgrading to CS 1.4/1.5? or references.
```

Comment:
The information were included in CS 1.4/1.5 release notes. I think this document does not need to have it.